analytics.js

```diff
@@ -307,6 +327,7 @@ class Analytics {
         const destConfig = intg.config;
         intgInstance = new intgClass(destConfig, self);
         intgInstance.init();
+        this.prepareDataForIntegrationsObj(intg, intgInstance);
```

Comment:
This may not be the right place to get the data from the integration. We have to wait for it to be loaded and ready.